pkg/distributor/distributor.go

```diff
@@ -938,6 +963,7 @@ func (d *Distributor) prePushValidationMiddleware(next push.Func) push.Func {
 
 			validatedSamples += len(ts.Samples)
 			validatedExemplars += len(ts.Exemplars)
+			validatedHistograms += len(ts.Histograms)
```

Comment:
This variable is now only used together with `validatedSamples`, so we can drop it.

Reply:
done